src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -112,6 +112,12 @@ public function load(array $configs, ContainerBuilder $container): void
         $patchFormats = $this->getFormats($config['patch_formats']);
         $errorFormats = $this->getFormats($config['error_formats']);
         $docsFormats = $this->getFormats($config['docs_formats']);
+        if (!$config['enable_docs']) {
+            // JSON-LD documentation format is mandatory, even if documentation is disabled.
+            $docsFormats = isset($formats['jsonld']) ? ['jsonld' => ['application/ld+json']] : [];
+            // If documentation is disabled, the Hydra documentation for all the resources is hidden by default.
+            $config['defaults']['hideHydraOperation'] = true;
```

Comment:
```suggestion
            $config['defaults']['hideHydraOperation'] = $config['defaults']['hideHydraOperation'] ?? true;
```

Thoughts? Or we should log something if the user configured this defaults?

Reply:
OK for this change.